arcgis-ios-sdk-samples/Edit data/Create mobile geodatabase/MobileGeodatabaseTableViewController.swift

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class MobileGeodatabaseTableViewController: UITableViewController {
+    // An array to hold all the features' OIDs.
+    var oidArray = [Int]()
+    // An array to store all the features' time stamps.
+    var collectionTimeStamps = [Date]()
```

Comment:
```suggestion
    var oidArray: [Int] = []
    // An array to store all the features' time stamps.
    var collectionTimeStamps: [Date] = []
```
As I [mentioned](https://github.com/Esri/arcgis-runtime-samples-ios/pull/1185#discussion_r923705434) earlier, Phil once mentioned about the style for type annotation, though I cannot find his comment.

Here is a reference to to the raywenderlich Swift Style Guide: https://github.com/raywenderlich/swift-style-guide/issues/143 and https://github.com/raywenderlich/swift-style-guide#type-inference